ext/standard/streamsfuncs.c

```diff
@@ -1636,9 +1636,6 @@ PHP_FUNCTION(stream_is_local)
 
 	if (Z_TYPE_P(zstream) == IS_RESOURCE) {
 		php_stream_from_zval(stream, zstream);
```

Comment:
Oh great this is a macro, that does macro things, and looks like a function, not at all suboptimal.

Reply:
Actually wait, this means there is a bug in the stubs here, as this function can return NULL...